octoprint_octorelay/__init__.py

```diff
@@ -38,7 +38,7 @@ class OctoRelayPlugin(
     def __init__(self):
         # pylint: disable=super-init-not-called
         self.polling_timer = None
-        self.tasks = [] # of Task
+        self.tasks: List[Task] = []
```

Comment:
From 3.9 can be just `list[Task]`, but in 3.8 have to use `typing`

https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#useful-built-in-types